src/ui/flutter_app/lib/l10n/intl_en.arb

```diff
@@ -272,25 +292,55 @@
     "@black": {
         "description": "Player 2"
     },
-    "loseReasonlessThanThree": " piece count is less than three.",
+    "loseReasonlessThanThree": "{loser} piece count is less than three.",
     "@loseReasonlessThanThree": {
-        "description": " piece count is less than three."
-    },
-    "loseReasonResign": " resign.",
+        "description": "Shows that the player lost because they had less than three pieces.",
```

Comment:
Show that the player lost because they had less than three pieces.

Reply:
"s" should not be added after "Show". I won’t point out the others one by one, and please replace them in full. Thank you!